assets/styles/style.css

```diff
@@ -0,0 +1,143 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("./assets/fonts/Roboto-Medium.ttf") format("truetype");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  position: relative;
+  height: 48px;
+  box-shadow: 0 8px 4px -4px rgba(128, 128, 128, 1);
+  /*It's actually much simpler, whatever you set the blur to (3rd value), set the spread (4th value) to the negative of it.*/
```

Comment:
And what about this comment?

Reply:
I just left it for myself (for possible future needs), because it cost me lots of nerves to dance around the box-shadow property. And when I found this genius and simple solution and decided to save it for the future.